Cargo.toml

```diff
@@ -33,3 +33,6 @@ axconfig = { git = "https://github.com/arceos-hypervisor/arceos.git", rev = "d93
 # Here, we use [patch] to unify `axtask` to the one from the `monolithic` branch of the arceos-org/arceos repository.
 [patch."https://github.com/arceos-hypervisor/arceos.git"]
 axtask = { git = "https://github.com/arceos-org/arceos.git", branch = "monolithic" }
+
+[patch."https://github.com/arceos-hypervisor/axvm.git"]
+axvm = { git = "https://github.com/aarkegz/axvm.git", rev = "3cedbae1" }
```

Comment:
It has been merged into master.
Update the url please.

Reply:
Okay

Reply:
Removed.